From f25f4465a6cc0d768c0e5afa1ef9b1fedcade760 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?=
 <nuclearsandwich@users.noreply.github.com>
Date: Mon, 11 Mar 2024 15:23:51 -0700
Subject: [PATCH] Override FetchContent flag to make sure that content is
 populated. (#10)

This flag now seems to be ON by default in our deb builds but our vendor
packages do not currently pre-populate fetchcontent contents.
---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index a856369d1..b36b206d9 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -35,6 +35,7 @@ override_dh_auto_configure:
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DAMENT_PREFIX_PATH="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DFETCHCONTENT_FULLY_DISCONNECTED=OFF \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.43.0

